### PR TITLE
Remove OS updates

### PR DESCRIPTION
**Pull request by danirabbit**
This doesn't completely remove packagekit because we use it for UbuntuDrivers

### CODE REVIEW

**Comment by danirabbit** [comment]
@leolost2605 I don't think runtime updates should be asking for a reboot right? Should we remove the restart notification completely?

**Comment by leolost2605** [comment]
@danirabbit I can't comment on runtime updates but AFAIK the only thing that would need a reboot are drivers and with elementary/settings-daemon#132 they can be removed as well so we might just remove it know 🤷

File: src/Core/UpdateManager.vala [hunk 1]
```diff
@@ -71,10 +58,6 @@ public class AppCenterCore.UpdateManager : Object {
             installed_package.update_state ();
         }
 
-
-        uint os_count = 0;
-        string os_desc = "";
-
 #if PACKAGEKIT_BACKEND
```
Comment (leolost2605) [inline src/Core/UpdateManager.vala:61]: I think we can remove all of this?

**Review by leolost2605 — CHANGES_REQUESTED** [review]
Looks good for now but I think we can remove even a bit more? For example we could just make packagekit return in cache refresh since it's not needed for ubuntu drivers i think?

**Review by leolost2605 — APPROVED** [review]
LGTM

**Pull request merged**